code/modules/mob/mob_grab.dm

```diff
@@ -159,9 +159,16 @@
 		affecting.Weaken(7)	//Should keep you down unless you get help.
 		if(ishuman(affecting))
 			var/mob/living/carbon/human/A = affecting
+			var/obj/item/clothing/C = A.head
+			if(C && (C.item_flags & THICKMATERIAL))
+				return
 			if(!(A.species.flags & NO_BREATHE))
 				A.losebreath = max(A.losebreath + 3, 5)
 				A.adjustOxyLoss(3)
+				if(affecting.stat == CONSCIOUS)
+					if(do_mob(assailant, affecting, 150))
+						A.visible_message(SPAN_WARNING("[A] falls unconscious..."), SPAN_DANGER("The world goes dark as you fall unconscious..."))
```

Comment:
```suggestion
						A.visible_message(SPAN_WARNING("[A] falls unconscious..."), FONT_LARGE(SPAN_DANGER("The world goes dark as you fall unconscious...")))
```

Reply:
should make it easier to see why you're paralyzed during combat

Reply:
You're not in combat if you got neck grabber for 30+ seconds